domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/BackwardsTimeTravelInStopNotice.java

```diff
@@ -27,7 +27,7 @@ public BackwardsTimeTravelInStopNotice(final String tripId,
                                            final String arrivalTimeAsString,
                                            final String previousStopDepartureTimeAsString,
                                            final Integer previousStopTimeStopSequence) {
-        super("stop_times.txt", E_047,
```

Comment:
Same here in terms of an unrelated change - was this intended, a bug fix, or an bad merge?

Reply:
Thanks, good catch I missed that one. It was intentional, so I'm removing this. Changes will be provided in a separate PR as previously discussed.